modules/installation-osp-configuring-sr-iov.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+[id="installation-osp-configuring-sr-iov_{context}"]
+= Creating compute machines that use SR-IOV hardware
+
+If your {rh-openstack-first} deployment supports link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html-single/network_functions_virtualization_planning_and_configuration_guide/index#assembly_sriov_parameters[single root I/O virtualization (SR-IOV)], you can provision an SR-IOV network that compute machines run on.
+
+
+.Prerequisites
+
+* Your cluster supports SR-IOV.
++
+[NOTE]
+====
+If you are unsure about what your cluster supports, review the {product-title} SR-IOV hardware networks documentation.
+====
+
+* You installed the SR-IOV Network Operator on your cluster.
```

Comment:
The SR-IOV Network Operator installation comes after the OCP cluster masters+workers are installed.

Reply:
@atyronesmith Would it make sense to list that as a "Next step," then, or should this module be completely separate from the installation flow?

Reply:
@atyronesmith ^^^

Reply:
Yes. I think that would be good.  Next steps to include the following...

1. Enable noiommu-vfio 
 kind: MachineConfig
apiVersion: machineconfiguration.openshift.io/v1
metadata:
  name: 99-vfio-noiommu
  labels:
    machineconfiguration.openshift.io/role: worker
spec:
  osImageURL: ''
  config:
    ignition:
      version: 2.2.0
    storage:
      files:
      - filesystem: root
        path: "/etc/modprobe.d/vfio-noiommu.conf"
        contents:
          source: data:text/plain;charset=utf-8;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
          verification: {}
        mode: 0644
2. Enable OpenStack metadata
kind: MachineConfig
apiVersion: machineconfiguration.openshift.io/v1
metadata:
  name: 20-mount-config
  labels:
    machineconfiguration.openshift.io/role: worker
spec:
  osImageURL: ''
  config:
    ignition:
      version: 2.2.0
    systemd:
      units:
        - name: create-mountpoint-var-config.service 
          enabled: true
          contents: |
            [Unit]
            Description=Create mountpoint /var/config
            Before=kubelet.service

            [Service]
            ExecStart=/bin/mkdir -p /var/config

            [Install]
            WantedBy=var-config.mount
          
        - name: var-config.mount
          enabled: true
          contents: |
            [Unit]
            Before=local-fs.target
            [Mount]
            Where=/var/config
            What=/dev/disk/by-label/config-2
            [Install]
            WantedBy=local-fs.target
3. Install SR-IOV Network Operator
see [SR-IOV Operator](https://docs.openshift.com/container-platform/4.6/networking/hardware_networks/installing-sriov-operator.html)

Reply:
@atyronesmith Is it fair to say that your steps 1 and 2 are RHOSP-specific prerequisites to that "Installing the SR-IOV Operator" page? 

If that's the case, I'd either:

* Roll them into a section of [post-installation network configuration](https://docs.openshift.com/container-platform/4.6/post_installation_configuration/network-configuration.html) that refers to the operator installation instructions at the end. 

So: Installation -> Post-installation configuration -> Operator installation 

_or_

* Duplicate the Operator installation page, add steps one and two at the top, and call it something like "Installing the SR-IOV Operator on RHSOP clusters" to have it all in one place.  

So: Installation -> RHOSP-specific operator installation

Reply:
@atyronesmith @iamemilio I noticed that vfio-noiommu and metadata aren't part of the upstream docs. Is there a gap there, or are those requirements documented along with operator installation docs somewhere?

Reply:
You are correct.  I forgot to add that documentation to the upstream docs.  I can work tomorrow on getting a PR to address steps 1 and 2.  I may miss code freeze for the upstream, but will have it there for GA.  Suggestions?

Reply:
I can't offer any advice about how to handle upstream--I don't typically work there. Emilio might have suggestions.

Reply:
Hi @maxwelldb, sorry, I was referring to going ahead with complete downstream document of the feature before it makes it upstream :).  

To answer the earlier question, yes, steps 1 and 2 are prerequisites for installing the SR-IOV operator in a shift on stack context.  I would suggest going with your first suggestion of a post-installation section.